### ISSUE TITLE
How do we need to slice tensor in forward pass for writing the custom loss function.

### ISSUE THREAD

**Issue by tiru1930** [tensorflow/tensorflow#33506]
I have created on a network for which input will be 

input shape is : (20, 16, 240, 320, 3)
output (logits is ) : Tensor("logits:0", shape=(?, 1), dtype=float32)
i,e  output in forward pass  for my network is 
`[array([[ -3.8974638 ],
       [ -5.2204075 ],
       [ -3.105205  ],
       [ -3.4843988 ],
       [ -4.0670795 ],
       [ -2.097713  ],
       [  2.6989975 ],
       [  3.1498132 ],
       [  2.3710475 ],
       [  2.0358462 ],
       [ -9.698921  ],
       [ -3.48162   ],
       [ -3.4569516 ],
       [ -2.9997227 ],
       [ -6.9699297 ],
       [-10.641993  ],
       [  2.6629193 ],
       [ -0.03444159],
       [ -1.6006284 ],
       [ -4.1530366 ]], dtype=float32)]
`
Now I want write loss functions on this logits as below mentioned 

loss = max(0,1-max(logits[0:10])+max(logits[10:20]) so that I can proceed for back prop.

How can I do this in  : tensorflow==1.14.0

**Comment by rmothukuru** [comment]
@tiru1930,
Can you please let us know what exactly you want to achieve using Custom Loss so that we can help you out. Thanks!

**Comment by tiru1930** [comment]
Eventually I am creating network to assign high score to anomaly videos and
low score to normal videos. So my model will take batch of anomaly and
normal videos at once and with loss function it assigned parameters
distinguish the anomaly and normal videos

**Comment by rmothukuru** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
